app/experimenter/experiments/tests/test_forms.py

```diff
@@ -1081,6 +1081,46 @@ def test_form_saves_risks(self):
         self.assertEqual(experiment.test_builds, data["test_builds"])
         self.assertEqual(experiment.qa_status, data["qa_status"])
 
+    def test_form_risk_technical_invariance(self):
```

Comment:
I don't think the word 'invariance' is descriptive here, the issue is that if risk_technical is true then risk_technical_description is required, so maybe something like `test_risk_technical_description_optionally_required` would be clearer.